frontend/src/services/planService.ts

```diff
@@ -19,7 +13,7 @@ export enum TaskState {
   VERIFIED_STATE = "verified",
 }
 
-export async function getPlan(): Promise<Plan | undefined> {
+export async function getPlan(): Promise<Task | undefined> {
```

Comment:
I found it a little weird that `getPlan()` returns a `Task`. Should it be `getTask()`?

Reply:
Yeah it's a little funky. A plan is a task, but a task is not a plan 🙃 

Basically "task" is a node in a tree-like data structure, and "plan" is the root node. But both Task and Plan have the same type/structure